### DISCUSSION TITLE
Indonesian PSE Certificate - Github could be blocked.

### DISCUSSION THREAD

**Discussion by TerribleSpeller** [community/community#27836] [Other Feature Feedback, Questions, & Ideas]
Hello, hopefully a staff member reads this, but I have to ask: Is Github going to register a Electronic System Provider license with the Indonesian Ministry of Communications and Information Technology? 

The government’s finally got off its chair and is enforcing a 
[Draconian law from 2020](https://www.permitindo.com/news/the-pse-what-is-it-do-i-need-it-how-do-i-get-it), which states that all electronic service providers must obtain a certificate from the government in order to operate in Indonesian cyberspace. Currently, several such electronic service providers like Steam, Paypal, and Godot Engine have been blocked (You can check on what is blocker and not here: https://kominfo-d.vercel.app/ ). And it seems the government will crack down on websites that haven’t registered within the following months. 

I really don’t want Github to be blocked, since lots of Indonesians working in the IT industry depends on Github like so many others. Hopefully Github can find a solution to this quickly.

Thanks, 
TerribleSpeller.

**Comment by jfullstackdev** [comment]
it's happening even in China, simply use VPN, 

but hopefully that will be dealt with by GitHub

**Comment by kunderemp** [comment]
> Hello, hopefully a staff member reads this, but I have to ask: Is Github going to register a Electronic System Provider license with the Indonesian Ministry of Communications and Information Technology?
> 

I hope Github would register as Electronic System Provider in Indonesia.  
In the news (in Indonesian Language) , General Directorate Information Application (Aptika) of the ministry, Semuel Abrijani Pangerapan
 said "GitHub will be included in the second wave (of the blockade) because it has paid service although it has free service. (However) we will evaluate again."
  
 
Reference:
[1] https://tekno.kompas.com/read/2022/08/02/13300047/kominfo-bakal-blokir-platform-digital-lain-yang-belum-daftar-pse-ada-github-?page=all.
 (in Indonesian language)
